### PR TITLE
Fading non-changing diff text.

### PR DESCRIPTION
**Pull request by BRevzin**
For me the green doesn't really stand out as much as I wanted it to, so I thought I'd just fade the background text a bit to make it pop.

### CODE REVIEW

File: data/template/wg21.html [hunk 1]
```diff
@@ -27,6 +27,7 @@
 $if(highlighting-css)$
   <style>
 $highlighting-css$
+code.diff {color: #$normcolor$}
```
Comment (mpark) [inline data/template/wg21.html]: For consistency what's currently generated, let's do:
```
code span. { color: #$uccolor$ }
```

File: data/metadata.yaml [hunk 2]
```diff
@@ -3,6 +3,7 @@ linkcolor: blue
 urlcolor: blue
 toccolor: black
 
+normcolor: 888888
```
Comment (mpark) [inline data/metadata.yaml]: Let's go with `uccolor` here for unchanged color. I feel like the "normal" thing makes sense in the context of the generic color scheme, but these colors are `diff` specific.

**Comment by mpark** [comment]
Looks good! I was gonna write the corresponding LaTeX change but
it's so simple that perhaps you can just include it in this patch?

We need to make the following diff to [wg21.py](https://github.com/mpark/wg21/blob/master/data/filter/wg21.py#L305-L310).

```diff
    return pf.Div(
+       pf.RawBlock(command.format('NormalTok', doc.get_metadata('uccolor')), 'latex'),
        pf.RawBlock(command.format('VariableTok', doc.get_metadata('addcolor')), 'latex'),
        pf.RawBlock(command.format('StringTok', doc.get_metadata('rmcolor')), 'latex'),
        result)
```

File: data/metadata.yaml (hunk 2 above)
Comment (mpark) [reply data/metadata.yaml]: Feels a bit silly asking this, but can we go with `898887` here?
It's the grey code from the `kate` highlighting color scheme.

Comment (BRevzin) [reply data/metadata.yaml]: Would be even sillier if I said no.

File: data/template/wg21.html (hunk 1 above)
Comment (mpark) [reply data/template/wg21.html]: Odd... what I suggested seems broken 🤔 Merging what you have.

Comment (mpark) [reply data/template/wg21.html]: Hm... weird: https://github.com/jgm/skylighting/blob/8c650b6e4aafc4c5eafb82d6e3449f3a67e066ee/skylighting-core/src/Skylighting/Format/HTML.hs#L49

**Pull request merged**